admin/server/organizations.go

```diff
@@ -199,6 +212,14 @@ func (s *Server) AddOrganizationMember(ctx context.Context, req *adminv1.AddOrga
 		return nil, status.Error(codes.PermissionDenied, "not allowed to add org members")
 	}
 
+	count, err := s.admin.DB.CountOrganizationOutstandingInvitations(ctx, org.ID)
+	if err != nil {
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+	if count >= org.QuotaOutstandingInvitations {
+		return nil, status.Errorf(codes.FailedPrecondition, "cannot invite more users, org %q already %d outstanding invitations", org.Name, org.QuotaOutstandingInvitations)
```

Comment:
Make error: `quota exceeded: org can at most have %d outstanding invitations`

Including the org name when it's in the request payload is not needed. That applies multiple places

Reply:
will change the error message. Default org name will get added to request most of the time without user entering it, so added just so that its in front of user.